.gitignore

```diff
@@ -436,6 +436,9 @@ cmake-build-*/
 
 test/types/string/validation/stress.good
 
+test/mason/masonGitDeps
+test/library/packages/UnitTest/Launcher_Test
```

Comment:
We want to be using directory-specific .gitignores, see https://github.com/chapel-lang/chapel/issues/20263

Reply:
Ah, I do remember seeing that at some point, thanks for reminding me. I've made that change.

Reply:
It's easy to forget, especially since we haven't had the time to remove the top-level one :)